docs/mkcloud.md

```diff
@@ -196,7 +216,7 @@ export cache_clouddata=1
 # setup/create lvm disk
 cloud_lvm_disk=$HOME/develcloud-lvm.raw
 if ! [ -f $cloud_lvm_disk ] ; then
-    qemu-img create -f raw $cloud_lvm_disk 100G
+    qemu-img create -f raw $cloud_lvm_disk 102G
```

Comment:
rationale for this change? why 102 and not 104?

Reply:
This was the minimum amount I needed to build a cloud with default options.

Reply:
> This was the minimum amount I needed to build a cloud with default options.

@cinerama70 with how many nodes? A minimum installation with 2 or even 3 nodes (and no ceph) should fit in 100GB.
Instead of changing it to 102GB rather mention that the required size depends on the cloud that should be deployed on that host (number of nodes, HA, ceph, ... might be factors).